sdcm/mgmt.py

```diff
@@ -314,6 +314,20 @@ class BackupTask(ManagerTask):
     def __init__(self, task_id, cluster_id, scylla_manager):
         ManagerTask.__init__(self, task_id=task_id, cluster_id=cluster_id, manager_node=scylla_manager)
 
+    def get_snapshot_tag(self, snapshot_index=0):
+        command = f" -c {self.cluster_id} task progress {self.id}"
+        res = self.sctool.run(command, parse_table_res=False)
+        if res.return_code != 0:
+            raise ScyllaManagerError(f"Failure for sctool '{command}' command:\n{res.stderr}")
```

Comment:
sctool.run already has an attribute named verify_errorless_result, which does exactly that